source/apps/deploy/automated.txt

```diff
@@ -0,0 +1,18 @@
+.. _automate-realm-app-deployment:
+
+=======================
+Automate App Deployment
+=======================
+
+.. default-domain:: mongodb
+
+.. toctree::
+   :titlesonly:
+   :caption: Automate App Deployment
+   :hidden:
+
+   Deploy Automatically with GitHub </apps/deploy/automated/deploy-automatically-with-github>
+   Set Up a CI/CD Pipeline </apps/deploy/automated/sdlc>
+
+- :doc:`Deploy Automatically with GitHub </apps/deploy/automated/deploy-automatically-with-github>`
+- :doc:`Set Up a CI/CD Pipeline </apps/deploy/automated/sdlc>`
```

Comment:
can you turn these to refs while you're in here mucking around?

Reply:
This page is gonna disappear in another PR. That said we use `:doc:` links consistently on container pages like this since it's a super easy copy paste from the toctree, unlike refs